src/main/java/com/epam/task/figures/data/DataReader.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.task.figures.data;
+
+import org.apache.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReader {
+    private static final Logger LOGGER = Logger.getRootLogger();
+
+    public List<String> readData(String filename) throws DataException{
+
+        try{
+            FileInputStream file = new FileInputStream(filename);
+            List<String> lines = new ArrayList<String>();
+            StringBuilder line = new StringBuilder();
+            int currentChar;
+            int size=0;
+
+            while((currentChar=file.read())!=-1){
```

Comment:
no need to do this low level stuff, just use BufferedReader
reader.readLine()
and close the reader in finally